lbrytv.docker.yml

```diff
@@ -2,5 +2,5 @@ StaticDir: /static
 BaseContentURL: https://api.lbry.tv/content/
 SentryDSN: "https://9ef75ef77a844d4aa35d834d78266477:3cff15dc36fa4595b056af781b042852@sentry.io/1416147"
 DatabaseConnection: postgres://postgres:postgres@postgres
-DatabaseName: lbrytv
+DatabaseName: postgres
```

Comment:
I think lbrytv is better. Why the change?

Reply:
The motivation behind the change is to keep the defaults. But you're right, a persistent database should be named.